README.md

```diff
@@ -150,7 +150,7 @@ const { flags, values, positionals } = parseArgs(argv, options);
 ### F.A.Qs
 
 - Is `cmd --foo=bar baz` the same as `cmd baz --foo=bar`?
-  - Yes, if `withValue: ['foo']`, otherwise no
+  - yes
```

Comment:
Test man here 🚨🙏🏻
could we add a test for this case?

I find very useful this kind of test:

```
test('Readme example', function(t) {
  const passedArgs1 = ['--foo=bar', 'baz'];
  const passedArgs2 = ['baz', '--foo=bar'];
  const expected = { flags: {}, values: { foo: 'bar' }, positionals: ['baz'] };
  const args = parseArgs(passedArgs1);

  t.deepEqual(args, expected, 'testing readme');
  t.deepEqual(args, parseArgs(passedArgs2), 'testing readme');

  t.end();
});
```

Reply:
Will do. If I put it in the middle I think it won't conflict with anything in flight...